tc/core/polyhedral/cuda/codegen.cc

```diff
@@ -681,6 +707,29 @@ size_t& nAstNodes() {
   return n;
 }
 
+namespace {
+// Collect ids of tensors that are only read on the Scop.
+std::unordered_set<isl::id, isl::IslIdIslHash> gatherReadOnlySet(
+    const MappedScop& mscop) {
+  std::unordered_set<isl::id, isl::IslIdIslHash> readOnlySet;
+
+  if (!mscop.useReadOnlyCache) {
+    return readOnlySet;
```

Comment:
I fixed this.

Reply:
I actually prefer returning a clearly empty container with `return {};`, otherwise you have to scan the code upwards to see there is nothing inside.

Reply:
Well, CI didn't like it.